### ISSUE TITLE
Webrecorder Contract - Phase 1

### ISSUE THREAD

**Issue by okdistribute** [bits-coop/projects#1]
# Autopilot Community Page: Initial Research

### How Ilya knows we're done
- [ ] Pull request with suggested copy for new Autopilot landing page focused on developers
- [ ] ​Pull requests on relevant repositories (https://github.com/webrecorder/behaviors​) improving the documentation for new contributors

### Steps to get there

- [ ] Research similar landing page sites for non-profit open source projects
	- https://delta.chat/en/ -> "Contribute"
    - 
- [ ] Look at 
- [ ] Create simple paper wireframe of what

**Comment by sedmonds** [comment]
@okdistribute thanks for initiating the issue.

In researching, community and documentation issues, current channel and repository support regarding the project appear to include: GitHub issues, tweets, email, or Gitter discourse.

Do we know if there is #channel on freenode for day to day communications or mailing list for community discussions and general question/news?

Of note, I did find some mention in r/DataHoarder.

**Comment by okdistribute** [comment]
@sedmonds good idea, we should definitely include that in our recommendations! #webrecorder on freenode with a bot that crosses with either matrix or slack is what people usually do

**Comment by sedmonds** [comment]
- [ ] Create simple copy/wireframe of what a Webrecorder Autopilot landing page would look like...

Contributors have four questions that require thoughtful answers. These questions might be:

1. How do I get started?
2. How do I modify and test an existing behavior?
3. How do I create a new behavior?
4. How do I test and deploy a new behavior?

Right now, a behavior is just a javascript file. No one is contributing new behaviors. Upon landing, I'd like to see how I would maintain, add, or fix a behavior if not working and, what behaviors currently are available.

@okdistribute I have a few #balsamiq mockups in queue.
